packages/flutter/test/widgets/scrollbar_test.dart

```diff
@@ -3240,20 +3240,191 @@ The provided ScrollController cannot be shared by multiple ScrollView widgets.''
     expect(scrollController.offset, 0.0);
```

Comment:
Is there a regression test for https://github.com/flutter/flutter/issues/150342?

Reply:
Good point, there isn't one. I will add it.